### DISCUSSION TITLE
New: updates!

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#465] [Show and tell] [answered]
Really nice that we can now post updates!!

![Screenshot 2021-10-29 15 43 42](https://user-images.githubusercontent.com/87815729/139445999-2aa76635-40e5-45a6-bae7-39bafac6c9a9.png)

**Comment by techsmyth** [comment, marked as answer]
Yeah! The first official update :) The honor will forever be yours @FloriSpoelstra 

![image](https://user-images.githubusercontent.com/30729240/139488847-1b9e9a07-2a54-401b-8c01-a1a2f19e1a46.png)

**Reply by FloriSpoelstra** [reply]
![wp5135729](https://user-images.githubusercontent.com/87815729/139523106-8ec2e666-13f2-45cc-9254-d286f71980a7.jpg)
